include/kmtest/kmtest.h

```diff
@@ -59,6 +59,12 @@
 #pragma section("KMTEST$__z", read)
 #pragma comment(linker, "/merge:KMTEST=.rdata")
 
+namespace
+{
+    DRIVER_OBJECT* s_driverObject = nullptr;
```

Comment:
Add `__declspec(selectany)` and put into kmtest namespace

Reply:
Removed the functions, because now the globals are available directly from the kmtest namespace